core/block_svg.js

```diff
@@ -1056,16 +1061,28 @@ Blockly.BlockSvg.prototype.updateCursor_ = function(e, closestConnection) {
   var showDeleteCursor = wouldDelete && !wouldConnect;
 
   if (showDeleteCursor) {
-    Blockly.Css.setCursor(Blockly.Css.Cursor.DELETE);
+    Blockly.utils.addClass(/** @type {!Element} */ (this.svgGroup_),
+                      'blocklyDraggingDelete');
+    
+    if (this.workspace.toolbox_) {
```

Comment:
Ah, good question. I was thinking that after your change, this would just go on at the beginning of a block drag, and be taken off at the end.  For now, I put it here since it was "safe" and because addClass only actually ads the style if it doesn't already exist so the performance characteristics aren't as bad as it initially looks.